juniorguru/web/templates/admin_jobs.html

```diff
@@ -70,6 +70,7 @@ <h2 id="{{ job.id }}">
       <summary>Data</summary>
       <pre>{{ job|pprint }}</pre>
     </details>
+    <div>Is Junior by Magical Leviathan:<strong>{{ job.magic_is_junior }}<a href="#">Report wrong classification</a></strong></div>
```

Comment:
Mám přísnou kontrolu nefunkčních kotev a odkazů (přece je to zhusta odkazovaný contentový web) a vztahuje se to i na admina, takže tohle bude failovat CI. Místo `<a href="#">` bude zatím potřeba použít `<span>` nebo nic, prostě to nemůže být nikam nemířící odkaz.